src/sentry/api/endpoints/organization_teams.py

```diff
@@ -86,7 +86,8 @@ def get(self, request, organization):
 
         :pparam string organization_slug: the slug of the organization for
                                           which the teams should be listed.
-        :param string detailed: Specify "0" to return team details that do not include projects
+        :param string detailed:      Specify "0" to return team details that do not include projects
+        :param string is_not_member: Specify "1" to return team details for teams that user is not a member of
```

Comment:
"to only return team details for teams that the user is not a member of" - I thought this was including teams the user isn't a member of as well as the teams they are a member of initially.